backend/chromeApiServer/src/main/java/com/canvas/service/helperServices/OAuthService.java

```diff
@@ -3,16 +3,25 @@
 import com.canvas.exceptions.CanvasAPIException;
 import com.fasterxml.jackson.databind.JsonNode;
 import okhttp3.Response;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
 
 /**
  * This class handles OAuth requests for generation, validation of access tokens
  */
 public class OAuthService {
 
     private final CanvasClientService canvasClientService;
+    private String clientSecret;
+    private String clientId;
     //TODO: Add and fetch values from config
-    private final String CLIENT_ID = "10000000000002";
-    private final String CLIENT_SECRET = "xppoeCamrsQVsEdZi5g6ykNE1qEJort6Zdn8ojGTcTBzRO15fR0AEdeeCtqcLymI";
+
+    @Autowired
```

Comment:
Add a no arg constructor with default variables we can use for testing and as a fall back.

Reply:
good point, will do

Reply:
I tried to add a no arg constructor but I don't understand how I can create a no arg constructor As I do not want to assign a hard coded value/ a null or empty value. Will address in a future PR. We have coverage for this so this should work for now.